.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+combine_as_imports = true
+include_trailing_comma = true
+line_length = 88
+multi_line_output = 3
+known_third_party = "pyotp,yaml"
```

Comment:
we aren't using these libraries, are we?

Reply:
removed